solr/core/build.gradle

```diff
@@ -115,6 +115,8 @@ dependencies {
 
   implementation 'commons-cli:commons-cli'
 
+  implementation 'net.jcip:jcip-annotations'
```

Comment:
So maybe @dsmiley  knows which `jcip-annotations` is correct.

At least based on https://github.com/stephenc/jcip-annotations - it should be https://mvnrepository.com/artifact/com.github.stephenc.jcip/jcip-annotations/1.0-1

however it looks like we include `net.jcip:jcip-annotations` already somehow?

Reply:
These two JARs are equivalent but have different licenses.  We should prefer `stephenc` as it's ASL licensed; a clean-room written implementation.  The  net.jcip https://jcip.net/annotations/doc/index.html is using Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.5) which the ASF is okay with in binary form https://www.apache.org/legal/resolved.html#cc-by but lets prefer the ASL one as we have a choice.

I removed net.jcip in #1313 this weekend; pending merge.

Reply:
@magibney lets remove the use of such annotations in this PR.  In #1313 I added it.

Reply:
👍 thanks for catching this, and for the work on #1313. I'll wait til after that's merged to merge this.